models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php

```diff
@@ -146,40 +112,51 @@ public function createContainer()
     /**
      * @inheritdoc
      */
-    public function add(TaskInterface $task, $status, $label = null)
+    public function add(TaskInterface $task, string $status, ?string $label = null): void
     {
         $this->getPersistence()->insert($this->getTableName(), [
             self::COLUMN_ID   => (string) $task->getId(),
             self::COLUMN_PARENT_ID  => $task->getParentId() ? (string) $task->getParentId() : null,
             self::COLUMN_TASK_NAME => $task instanceof CallbackTaskInterface && is_object($task->getCallable()) ? get_class($task->getCallable()) : get_class($task),
             self::COLUMN_PARAMETERS => json_encode($task->getParameters()),
             self::COLUMN_LABEL => (string) $label,
-            self::COLUMN_STATUS => (string) $status,
+            self::COLUMN_STATUS => $status,
             self::COLUMN_OWNER => (string) $task->getOwner(),
             self::COLUMN_CREATED_AT => $task->getCreatedAt()->format($this->getPersistence()->getPlatForm()->getDateTimeFormatString()),
             self::COLUMN_UPDATED_AT => $this->getPersistence()->getPlatForm()->getNowExpression(),
-            self::COLUMN_MASTER_STATUS => (int) $task->isMasterStatus(),
-        ]);
+            self::COLUMN_MASTER_STATUS => $task->isMasterStatus(),
+        ], $this->getTypes());
     }
 
-    /**
-     * @inheritdoc
-     */
-    public function getStatus($taskId)
+    protected function getTypes(array $data = []): array
     {
-        $qb = $this->getQueryBuilder()
-            ->select(self::COLUMN_STATUS)
-            ->from($this->getTableName())
-            ->andWhere(self::COLUMN_ID . ' = :id')
-            ->setParameter('id', $taskId);
+        return [
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::STRING,
+            ParameterType::BOOLEAN,
+        ];
+    }
 
-        return $qb->execute()->fetchColumn();
+    public function __toPhpCode()
+    {
+        return 'new ' . get_called_class() . '('
+            . \common_Utils::toHumanReadablePhpString($this->persistenceId)
+            . ', '
+            . \common_Utils::toHumanReadablePhpString($this->containerName)
+            . ')';
     }
 
     /**
      * @inheritdoc
      */
-    public function updateStatus($taskId, $newStatus, $prevStatus = null)
+    public function updateStatus(string $taskId, string $newStatus, ?string $prevStatus = null): int
```

Comment:
```suggestion
    public function updateStatus(string $taskId, string $newStatus, string $prevStatus = null): int
```

Default value already provided